### ISSUE TITLE
Immediately prompt for touch id if enabled

### ISSUE THREAD

**Issue by markusait** [MacPass/MacPass#1306]
**Is your feature request related to a problem? Please describe.**
I'm on the branch with Touch ID support (not sure if it's already on main/master). And it's a hassle to have to use the mouse each time to click the touch ID button to be prompted to authenticate via Touch ID.
https://github.com/MacPass/MacPass/issues/514

**Describe the solution you'd like**
Immediately prompt for TouchID on lock page

**Describe alternatives you've considered**
Allow to open Touch ID auth via a hotkey/keyboard shortcut

**Comment by mstarke** [comment]
There are some things that need to be considered when adding this:

* If you have multiple databases opened at once, should MacPass request touchID for all of them?
* If MacPass re-locks the database, what is the indicator to re-request touchID?
* TouchID might not work since the credentials are wrong, this then requires a fallback workflow
* Should the user be able to enable/disable this feature? Does this then work application wide or per database?

**Comment by markusait** [comment]
Hmm good point. I'd say just add a shortcut for being able to prompt for touch id then instead of auto-prompting

**Comment by orestesgaolin** [comment]
Can you try this build? I made this change for myself and it seems to work well https://github.com/orestesgaolin/MacPass/releases/tag/0.8.4-rc3

**Comment by mstarke** [comment]
@orestesgaolin Can you dive in to the details on how you handled the issues raised in https://github.com/MacPass/MacPass/issues/1306#issuecomment-2467631340

**Comment by orestesgaolin** [comment]
Yes, here's the commit https://github.com/orestesgaolin/MacPass/commit/c51a7d27e828f5419564a92ed467389b9fa3a609

main reasoning was `attemptTouchID` via new callback when touch id is enabled; this triggers the prompt

https://github.com/user-attachments/assets/a6fde621-d0f6-4def-a0e7-b69d6b6743de
